### DISCUSSION TITLE
Base image layer vulnerabilities

### DISCUSSION THREAD

**Discussion by TimBrown1611** [aquasecurity/trivy#7185] [Q&A]
### Question

Does trivy knows to separate base image vulnerabilities from other layers vulnerabilities?

### Target

Container Image

### Scanner

Vulnerability

### Output Format

JSON

### Mode

Standalone

### Operating System

macOS

**Comment by knqyf263** [comment]
Duplicate of https://github.com/aquasecurity/trivy/issues/1198
